### PR TITLE
ZoomIn and ZoomOut buttons options

### PR DESCRIPTION
**Pull request by xgeo**
An extra option for interface

### CODE REVIEW

File: src/image-viewer.component.ts [hunk 1]
```diff
@@ -39,6 +42,8 @@ export class ImageViewerComponent implements OnChanges, OnInit, AfterViewInit {
     @Input() rotateLeftTooltipLabel = 'Rotate left';
     @Input() resetZoomTooltipLabel = 'Reset zoom';
     @Input() fullscreenTooltipLabel = 'Fullscreen';
+    @Input() zoomInTooltipLabel = 'Zoom In';
+    @Input() zoomOutTooltipLabel = 'Zoom In';
```
Comment (brenoprata10) [inline src/image-viewer.component.ts]: Please, Correct the corresponding label to 'Zoom Out'

**Review by brenoprata10 — COMMENTED** [review]
Please, Correct the corresponding label to 'Zoom Out'

File: src/image-viewer.component.ts (hunk 1 above)
Comment (xgeo) [reply src/image-viewer.component.ts]: done, thanks,

**Comment by brenoprata10** [comment]
Approved, thanks for contributing o/

**Pull request merged**